src/components/ModalConfirmClearStorage.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { t } from 'ttag';
+import $ from 'jquery';
+import SpanFmt from './SpanFmt';
+import { CONFIRM_RESET_MESSAGE } from '../constants';
+
+export default function ModalConfirmClearStorage({ onClose, success }) {
+  const [confirmText, setConfirmText] = useState('');
+  const [confirmError, setConfirmError] = useState('');
+
+  useEffect(() => {
+    $('#modalConfirmResetAllData').modal('show');
+    $('#modalConfirmResetAllData').on('hidden.bs.modal', onClose);
+
+    return () => {
+      $('#modalConfirmResetAllData').modal('hide');
+      $('#modalConfirmResetAllData').off();
+    };
+  }, []);
+
+  const confirmResetData = useCallback(() => {
+    if (confirmText.toLowerCase() !== CONFIRM_RESET_MESSAGE.toLowerCase()) {
+      setConfirmError(t`Invalid value.`);
+      return;
+    }
+
+    success();
+  }, [confirmText, setConfirmError]);
+
+  return (
+    <div className="modal fade" id="modalConfirmResetAllData" tabIndex="-1" role="dialog" aria-labelledby="alertModal" aria-hidden="true" data-backdrop="static" data-keyboard="false">
+      <div className="modal-dialog" role="document">
+        <div className="modal-content">
+          <div className="modal-header">
+            <h5 className="modal-title">
+              { t`Confirm reset all data` }
+            </h5>
+          </div>
+          <div className="modal-body">
+            <div>
+              <p><SpanFmt>{t`**Make sure you have the backup of your seed because this will turn your wallet into a fresh install.**`}</SpanFmt></p>
+              <p>{t`Do you want to reset all of your wallet data? Only continue if you know what you are doing.`}</p>
+              <p>{t`This action cannot be undone. All your data will be erased.`}</p>
+              <p>{t`Your wallet uniqueId will be reset.`}</p>
+              <p>uniqueId: {localStorage.getItem('app:uniqueId')}</p>
+              <p>{t`If you want to reset all data, please type '${CONFIRM_RESET_MESSAGE}' in the box below and click on 'Reset all data' button.`}</p>
```

Comment:
Does it make sense to add one more sentence like this?

```
<p>{t`After resetting the data, the application will close, and you will need to open it again.`}</p>
```

Reply:
Done